.github/workflows/build-deps.yml

```diff
@@ -178,13 +194,20 @@ jobs:
     - uses: actions/download-artifact@v3
       with:
         name: llvm-arm64.zip
+
+    - uses: actions/download-artifact@v3
+      with:
+        name: node-arm64.zip
         
     - name: Build emsdk arm64
       run: |
         ls
         New-Item -ItemType Directory -Force -Path emsdk/emscripten
         Expand-Archive -LiteralPath emscripten_package.zip -DestinationPath emsdk\emscripten
         Expand-Archive -LiteralPath llvm-arm64.zip -DestinationPath emsdk
+        Expand-Archive -LiteralPath node-arm64.zip -DestinationPath emsdk\node
```

Comment:
Build emsdk arm64 was supposed to be used for llvm emscripten package as part of emscripten_build job. I suggest to create separate job for emsdk release which will have dependencies on  emscripten_build and build-nodejs-arm64